---
title: Teorica Nueve
icon: fontawesome/solid/brain
tags: 
  - teoricas
---

## Secuenciación y ensamblado de genomas 
 Clase teórica sobre métodos de secuenciación, ensamblado de genomas, mapeo de lecturas contra referencia, etc. Valores de calidad de base, cálculo de coberturas, algoritmos.

 
 * :fontawesome-regular-file-pdf: [Slides](genomeSequencingAssemblyReadMapping2025.pdf) 
 * :material-video-check: [Video grabado en 2023.](https://youtu.be/tIDj8FMeNSQ) 
 

## Material de lectura y consulta

  * :fontawesome-regular-file-pdf: Compeau PEC, Pevzner PA, Tesler G. How to apply de Bruijn graphs to genome assembly. Nature Biotechnol 29: 987, 2011. [PDF de estudio.](compeau_11_debruijn.pdf)
  * :fontawesome-regular-file-pdf: Nagarajan, N., & Pop, M. (2013). Sequence assembly demystified. Nature Reviews Genetics, 14(3), 157–167. doi:10.1038/nrg3367. [PDF de estudio.](nagarajan2013.pdf)
  * :fontawesome-regular-file-pdf: Rice, E. S., & Green, R. E. (2018). New Approaches for Genome Assembly and Scaffolding. Annual Review of Animal Biosciences, 7(1). doi:10.1146/annurev-animal-020518-115344. [PDF de estudio.](rice2018.pdf)
  * :paperclip: Ribeiro FJ, Przybylski D, Yin S, et al. Finished bacterial genomes from shotgun sequence data. Genome Res. 2012 Nov;22(11):2270-7. [doi:10.1101/gr.141515.112](https://doi.org/10.1101/gr.141515.112). PMID: 22829535; PMCID: PMC3483556.
  * :material-wikipedia: [Phred Quality Score](https://en.wikipedia.org/wiki/Phred_quality_score)
  * :material-wikipedia: [K-mers](https://en.wikipedia.org/wiki/K-mer)  
  * :material-wikipedia: [N50, L50 and related statistics](https://en.wikipedia.org/wiki/N50,_L50,_and_related_statistics)

## Software 

 * :material-download: [FASTQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
 * :material-download: [MultiQC](https://multiqc.info/)
 * :material-download: [KAT: K-mer Analysis Toolkit](https://github.com/TGAC/KAT)
